README.md

```diff
@@ -112,7 +112,7 @@ To show some simple examples (from
 cy.queryByText('Button Text').should('exist')
 cy.queryByText('Non-existing Button Text').should('not.exist')
 cy.queryByLabelText('Label text', {timeout: 7000}).should('exist')
-cy.findAllByText('Jackie Chan').click({multiple: true})
+cy.findAllByText('Jackie Chan').eq(0).click();
```

Comment:
`.eq(0)` isn't necessary unless multiple are found.  I'd rather not make people think they have to add `.eq(0)` because they are using a `*All*` query and wanting to click.

Thoughts? I have to resolve a conflict on this line, but I don't want to bulldoze over this change with my opinions.

Reply:
Good point. Feel free to clarify in what ever way makes the most sense 👍 Thanks!